core/modules/admin_bar/admin_bar.theme.inc

```diff
@@ -22,6 +22,16 @@ function theme_admin_bar_icon_page($variables) {
  return t('This page');
 }
 
+/**
+* Renders a language icon to display in the administration bar.
```

Comment:
I'm not happy with the description of this function here. It seems to be returning text - not an icon. So it is inaccurate. ...or am I missing something? 🤔 

We seem to be mimicking the descriptions of the 2 functions before it, so lets please fix things.

Also, the function immediately before this seems to be missing an indentation space ...wondering why PHPCS didn't pick that when it's only 2-3 lines above 🤔 ...in any case, can we please fix that here as well?

Reply:
I suggest you open a new issue for this. I feel like this PR keeps dragging on and on. I don't have much time before freeze to keep making changes to this.

Reply:
Done: https://github.com/backdrop/backdrop-issues/issues/6091